pyiron_atomistics/sphinx/base.py

```diff
@@ -1890,10 +1890,7 @@ class Output:
 
     def __init__(self, job):
         self._job = job
-        self.generic = DataContainer(table_name="output/generic")
-        self.charge_density = SphinxVolumetricData()
-        self.electrostatic_potential = SphinxVolumetricData()
-        self.generic.create_group("dft")
+        self.clear_output_for_restart()
```

Comment:
```suggestion
        self.generic = None
        self.charge_density = None
        self.electrostatic_potential = None
        self.clear_output_for_restart()
```
Some code linters complain when the variables are not initialised in the `__init__()` function.